packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -513,6 +520,7 @@ export class DatabaseCluster extends DatabaseClusterNew {
       // Encryption
       kmsKeyId: props.storageEncryptionKey?.keyArn,
       storageEncrypted: props.storageEncryptionKey ? true : props.storageEncrypted,
+      copyTagsToSnapshot: props.copyTagsToSnapshot,
```

Comment:
So, right now, in `DatabaseInstance`, this defaults to `true`.

I wonder if we can safely set this to `true` here too? It would be a template-level change, but I wonder whether that's OK, because it's a safe one? I see that [the property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-copytagstosnapshot) has "Update Type: no interruption", which is a good sign.

Can you do an experiment, and confirm changing this on an existing Cluster from "not specified" to `true` is a safe operation? If it is, I think it would be fine to default this to `true` here.